src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1332,17 +1341,29 @@ public void testDefaultLoggerClosesItStreams() throws Exception {
 
             verify(checker, tmpFile.getPath(), expected);
 
-            assertEquals("Close count was not expected",
-                    1, testInfoOutputStream.getCloseCount());
-            assertEquals("Flush count was not expected",
-                    3, testInfoOutputStream.getFlushCount());
-            assertEquals("Close count was not expected",
-                    1, testErrorOutputStream.getCloseCount());
-            assertEquals("Flush count was not expected",
-                    1, testErrorOutputStream.getFlushCount());
+            assertWithMessage("Close count")
+                    .that(testInfoOutputStream.getCloseCount())
+                    .isEqualTo(1);
+            assertWithMessage("Flush count")
+                    .that(testInfoOutputStream.getFlushCount())
```

Comment:
I see 2 sets of close/flush count. Can you expand the text to show the differences.
IE: Error close count, Output close count, etc...

Reply:
done